ICSharpCode.ILSpyX/Analyzers/ExportAnalyzerAttribute.cs

```diff
@@ -31,5 +34,17 @@ public ExportAnalyzerAttribute() : base("Analyzer", typeof(IAnalyzer))
 		public required string Header { get; init; }
 
 		public int Order { get; set; }
+
+		public static IEnumerable<(ExportAnalyzerAttribute AttributeData, Type AnalyzerType)> GetAnnotatedAnalyzers()
+		{
+			foreach (var type in typeof(ExportAnalyzerAttribute).Assembly.GetTypes())
+			{
+				var exportAnalyzerAttribute = type.GetCustomAttribute(typeof(ExportAnalyzerAttribute), false) as ExportAnalyzerAttribute;
+				if (exportAnalyzerAttribute is not null)
```

Comment:
if (type.GetCustomAttribute(typeof(ExportAnalyzerAttribute), false) is ExportAnalyzerAttribute exportAnalyzerAttribute)